### ISSUE TITLE
Npm run hot fails with copy ()

### ISSUE THREAD

**Issue by skyronic** [JeffreyWay/laravel-mix#313]
Sample mix config:

This works with `npm run hot`

```
mix.js('resources/assets/js/app.js', 'public/js/app.js')
   .sass('resources/assets/sass/app.scss', 'public/css/app.css')
```

This doesn't work with `npm run hot` and fails with unspecified error

```
mix.js('resources/assets/js/app.js', 'public/js/app.js')
   .sass('resources/assets/sass/app.scss', 'public/css/app.css')
   .copy ('resources/tmp', 'public/tmp'); 
```

```
 ERROR  Failed to compile with 1 errors

 error
```

**Comment by ArjanSchouten** [comment]
OS: Windows 10
Nodejs: v6.9.5
Yarn: yes
Npm: yes

I have this issue too.
@skyronic which OS are you using? It might be a Windows problem?!

Running npm run dev does work properly. I already tried to copy single files and directories.
Also when I try to both relative and absolute paths it simply does not work.

The error message is not descriptive at all.
Does anyone has a clue where the error is comming from?

**Comment by skyronic** [comment]
@ArjanSchouten I am on Mac OS. Yeah, copying works great on `npm run dev`. Currently I'm using mix for hot reload and gulp for copying. HMR and Mix's speed is totally worth the temporary inconvenience though.

**Comment by terrylow** [comment]
facing the exact error with `npm run hot` too

**Comment by AlfredoHernandez** [comment]
In my current project, also fails with `npm run dev`, like @skyronic

**Comment by yazfield** [comment]
same problem

**Comment by TiBianMod** [comment]
>OS: macOS Sierra 10.12.4
>Nodejs: v7.9.0
>Laravel-Mix: v0.10.0

same problem here with `npm run hot`.

Both methods `mix.copy()` & `mix.copyDirectory()` reproduce an error.

**Comment by vaniyokk** [comment]
+1, same issue. Windows 10x64, node 6.9.5

**Comment by TiBianMod** [comment]
#63

**Comment by TiBianMod** [comment]
> Temporary solution disable mix.copy when you running `npm run hot`

```js
if (! mix.config.options.hmr) {
    mix.copy('path/from', 'path/to');
}
```

**Comment by JeffreyWay** [comment]
Fixed in 329ed0ccffb51d52111ec11359f2617c528c60c0.
